db/re/item_db_equip.yml

```
@@ -98668,9 +98668,7 @@ Body:
     Script: |
       bonus bFlee,5;
       .@r = getrefine();
-      if (.@r>=2) {
-         bonus bCritical,2;
-      }
+      bonus bFlee,.@r/2;
```

Delete the previous bFlee,5 and
```suggestion
      bonus bFlee,5+.@r/2;
```

---

Thanks